week7/tsmessage/src/types/common.ts

```diff
@@ -0,0 +1,6 @@
+export interface List {
```

Comment:
```suggestion ts
export interface LetterInfo
```
로 조금더 인터페이스명이 명확했음 좋겠다! 
그리구 types/common.ts로 했는데, 여기서는 공통으로 타입이 사용되는게 맞지만 만약 기능이 많다면, `types/letter.ts` 로 파일명 지정하는게 관리하기 편할거야!